build-support/common.sh

```diff
@@ -67,3 +67,15 @@ function git_merge_base() {
   git rev-parse --symbolic-full-name --abbrev-ref HEAD@{upstream} 2>/dev/null || echo 'master'
 
 }
+
+function validate_executable() {
+  local -r exe_name="$1"
+  # `command -v` is the POSIX way to check for a command's existence -- see
+  # https://stackoverflow.com/a/677212/2518889.
```

Comment:
We're going to end up using this idiom over 5 times in our codebase, at which point I think the comment becomes unnecessary. The context of this function makes this code self-documenting enough, imo. Also, if there is lingering confusion, it's easy to search what `command -v` does.

With the Shellcheck lint coming soon, we also don't need this comment to ensure people don't revert and use `which` or `hash`, because Shellcheck won't let them.